bcipy/gui/viewer/qt_data_viewer.py

```diff
@@ -0,0 +1,585 @@
+"""EEG Data Viewer"""
```

Comment:
Most of the functionality related to managing data and rendering using Matplotlib is identical to the wxpython version thanks to the multiple backends supported by Matplotlib. The main differences are in the widget creation (see `initUI`), events, and timer.